ansible/roles/test/tasks/pfc_wd/functional_test/functional_test.yml

```diff
@@ -33,7 +33,7 @@
   set_fact:
       pfc_wd_detect_time: 200
       pfc_wd_restore_time: 200
-      pfc_wd_restore_time_large: 30000
+      pfc_wd_restore_time_large: 50000
```

Comment:
Current test only run for one port and 50s is fine. Since we are going to iterate all the ports, waiting for such long time will make test run for hours as each port will wait around 2mins simply for this every time. We will have to make this parameter small in future.

Reply:
the value of 50s was approved by @marian-pritsak :)

Reply:
50s is too large for iterate all the ports as above reason. We could set it to 50s for now. Please be aware that we need change it to around 3-5s once we enable the test over all the ports @marian-pritsak